lib/controller.dart

```diff
@@ -98,9 +104,10 @@ void loadDataFromFirebase() async {
   }
 }
 
+// Compute data methods
 Map<String, Set> computeUniqFieldValues(List<model.Filter> filterOptions,
     Map<String, Map<String, dynamic>> interactions) {
-  var uniqueFieldValues;
+  var uniqueFieldValues = Map<String, Set>();
```

Comment:
This is supposed to go into the prev PR, realising it much later

Reply:
ah, no worries, I've also left a comment on the other PR, sorry about the cross interaction!

Since this PR is on top of the other one, no need to also address it there, it'll get done when merging